server/content/establishmentData.json

```diff
@@ -3,7 +3,6 @@
     "name": "berwyn",
     "formattedName": "Berwyn",
     "prefix": "HMP",
-    "uuId": "fd1e1db7-d0be-424a-a3a6-3b0f49e33293",
```

Comment:
these UUIDs were used to filter categories but the new api filters by prison on the drupal side